src/static/styles/partials/components/_wf-tools-devtools.scss

```diff
@@ -1,52 +1,134 @@
-.wf-tools-panel {
```

Comment:
OK I just used what was already there. I've moved it to the top level tools stylesheet instead. There's no other page that puts styles in those files but I agree it makes sense.